src/Controller/CRUDController.php

```diff
@@ -212,7 +212,7 @@ public function batchActionDelete(ProxyQueryInterface $query)
      *
      * @return Response|RedirectResponse
      */
-    public function deleteAction($id)
+    public function deleteAction($id = null)
     {
         $request = $this->getRequest();
         $id = $request->get($this->admin->getIdParameter());
```

Comment:
No change here?

Reply:
What do you expect ?

Reply:
I would expect, that we don't need to call `$request->get($this->admin->getIdParameter());`, because we use the same magic like symfony does when resolving route parameters. This call looks like a hack to me, because the routing does not work as expected.

Reply:
I find
```
$id = $request->get($this->admin->getIdParameter());
```

In all the deleteAction, editAction, showAction, historyAction, historyViewRevisionAction, historyCompareRevisionsAction, aclAction.

Adding `= null` fixed a bug.
I really don't know what's the impact of removing `$id = $request->get($this->admin->getIdParameter());` what's why I didn't. If we remove it from one action, we should remove it from all the action ; I can, but as said I really don't know what's the impact.

Reply:
I would look at the git history to try to find since when we have this weird situation. Maybe you will find an explanation.

Reply:
I looked at.
It's here since the first implementation of deleteAction. Same for others actions.

Reply:
This sounds fishy :sweat_smile:

Reply:
Ah the skeleton was created before the implementation, this looks like a careless mistake: 5109228ab7f5c0f285888ce85162b60fffb1a9e0